app/src/main/java/org/simple/clinic/overdue/communication/CommunicationPullResponse.kt

```diff
@@ -10,7 +10,7 @@ data class CommunicationPullResponse(
     @Json(name = "communications")
     override val payloads: List<CommunicationPayload>,
 
-    @Json(name = "processed_since")
+    @Json(name = "process_token")
     override val processedSinceTimestamp: String
```

Comment:
We could rename the variable `processedSinceTimestamp` in all our Response models.
Also in `DataPullResponse`.

Reply:
Makes sense, I'll do this.

Reply:
Done.